app/uk/gov/hmrc/trackingconsentfrontend/assets/src/domain/userPreferencesFactory.ts

```diff
@@ -50,15 +60,15 @@ const userPreferencesFactory = (): UserPreferences => {
     return entries.filter(isPreferenceStrictlyBoolean)
   }
 
-  const allThePreferences = () => fromEntries(cookieTypes.map(cookieType => [cookieType, true]))
+  const allThePreferences = () => fromEntries(cookieTypes.map((cookieType) => [cookieType, true]))
 
-  const getPreferences = () => {
+  function getPreferences(): Preferences | undefined {
     const cookie = Cookies.getJSON(COOKIE_CONSENT)
     if (cookie === null || cookie === undefined || cookie.preferences === undefined) {
       return undefined
     }
 
-    const { version } = cookie
+    const { version, datetimeSet } = cookie
```

Comment:
It looks like this new variable is never used.  Can it be removed?

Reply:
Good catch, removed.